var/spack/repos/builtin/packages/mesa/package.py

```diff
@@ -114,13 +114,17 @@ class Mesa(MesonPackage):
         depends_on("libllvm@6:")
         depends_on("libllvm@:11", when="@:20")
         depends_on("libllvm@:12", when="@:21")
+    depends_on("unwind", when="+libunwind")
```

Comment:
There's a dependency on `unwind` at line 5-. The variant `libunwind` doesn't seem to be defined at all.